### ISSUE TITLE
Ingest DO, ORDO, OMIM, NCIT mappings

### ISSUE THREAD

**Issue by matentzn** [mapping-commons/disease-mappings#5]
- [ ] Have separate goals for obtaining the sources, which may eventually be extended to contain preprocessing (e.g. `mirror/do.owl:`)
- [ ] Using sssom-py extract all mappings from DO, ORDO, OMIM and NCIT.
- [ ] Manually review a dozen or so in the source ontology whether they don't contain extract curation information (ORDO `ntbt` for example)

**Comment by matentzn** [comment]
- [ ] Can you double-check that ncit.owl contains the ICD 10 mappings mentioned here:
https://ncit.nci.nih.gov/ncitbrowser/pages/mapping_search.jsf?nav_type=mappings

**Comment by hrshdhgd** [comment]
You mean [NCIt_Maps_To_ICD10_2016 (22.01e)](https://ncit.nci.nih.gov/ncitbrowser/ajax?action=export_maps_to_mapping&target=ICD10%202016) and [NCIt_Maps_To_ICD10CM_2017 (22.01e)](https://ncit.nci.nih.gov/ncitbrowser/ajax?action=export_maps_to_mapping&target=ICD10CM%202017) ?

**Comment by matentzn** [comment]
Yeah, maybe.. Not sure. I guess I am wondering whether ncit.owl contains the mappings mentioned on this page, and if the page is mentioning these two, then yes. 

I would love these mappings, especially the newest one, to be included here!

**Comment by hrshdhgd** [comment]
So ncit.owl has just the mention of `ICD10 2017` in them. Does not seem to have the actual ICD10 code. Although the mapping tables linked above have the mapping: 800 (ICD10 2017), 15 (ICD10 2016) and have the corresponding NCIT code.

**Comment by matentzn** [comment]
Perfect, thank you for checking. In this case, let's build an ingest into SSSOM for these!

**Comment by hrshdhgd** [comment]
So do you mean having a new method in `sssom-py` that accommodates for external mapping files?

**Comment by matentzn** [comment]
Big sure where to draw the line. Maybe for now a simple python script that imports SSSOM mapping and mapping sets and just do it externally? I told @joeflack4 to add the ICD10-snomed mapping parser to SSSOM py but I am not sure. Maybe yes, adding it to parsers would be an option. For now the answer is: python, in whatever way you like. But perhaps having a method from_ncit() in parsers.py would be awesome.

**Comment by hrshdhgd** [comment]
Alright, for the time being I'll just do an external `JOIN` so to speak to include it in the SSSOM mappings as a quick-n-dirty solution. We can discuss a generic solution down the road after having experience with the different scenarios.

**Comment by matentzn** [comment]
Sounds good!

**Comment by hrshdhgd** [comment]
ICD mapping column:

RELATIONSHIP_TO_TARGET
--
Broader Than
Related To
Has Synonym
Narrower Than

Which one would be appropriate?

```
"Broader Than": "skos:broader",
"Related To": "skos:relatedMatch",
"Has Synonym": "skos:altLabel",
"Narrower Than": "skos:related",
```
or
```
"Broader Than": "oboInOwl:hasBroadSynonym",
"Related To": "oboInOwl:hasRelatedSynonym",
"Has Synonym": "oboInOwl:hasExactSynonym",
"Narrower Than": "oboInOwl:hasNarrowSynonym",
```

**Comment by matentzn** [comment]
It's confusing but bare with me:

- Broader than: skos:narrowMatch
- Narrower than skos:broadMatch

**Comment by matentzn** [comment]
Related to : skos:relatedMatch

We don't need synonym unless the object of synonym is a code and not a label. What about "equivalent"?

**Comment by matentzn** [comment]
Maybe synonym is equivalent

**Comment by hrshdhgd** [comment]
skos:closeMatch, skos:exactMatch, owl:sameAs, owl:equivalentClass, owl:equivalentProperty from https://www.w3.org/TR/skos-reference/#L4858

**Comment by matentzn** [comment]
Use skos:exactMatch in these cases

**Comment by matentzn** [comment]
https://thesaurus.cancer.gov/ncitbrowser/pages/mapping_search.jsf?nav_type=mappings&b=0&m=0
